homeassistant/components/zwave_js/triggers/value_updated.py

```diff
@@ -23,16 +29,7 @@
     ATTR_PROPERTY_NAME,
     DOMAIN,
 )
-from homeassistant.components.zwave_js.helpers import (
-    async_get_nodes_from_targets,
-    get_device_id,
-)
-from homeassistant.const import ATTR_DEVICE_ID, ATTR_ENTITY_ID, CONF_PLATFORM, MATCH_ALL
-from homeassistant.core import CALLBACK_TYPE, HassJob, HomeAssistant, callback
-from homeassistant.helpers import config_validation as cv, device_registry as dr
-from homeassistant.helpers.trigger import TriggerActionType, TriggerInfo
-from homeassistant.helpers.typing import ConfigType
-
+from ..helpers import async_get_nodes_from_targets, get_device_id
 from .helpers import async_bypass_dynamic_config_validation
```

Comment:
This is a bit confusing. Having both `helpers` files in both the current and the parent dir.
Maybe `triggers/helpers.py` can be renamed? Or it might even be possible to merge the contents in the parent `helpers`.

Reply:
I agree that it is confusing... but should it hold up the PR?
Maybe someone from zwave_js team can adjust in follow up?
Or do you think it needs to be preliminary?

Reply:
I'll open a followup to rename `triggers/helpers.py`.

Reply:
Opened #78331